### PR TITLE
Ram  add majors command

### PR DESCRIPTION
**Pull request by ramsgoli**
## Description
Populate the database with majors

## Pls do 
no
- [x] Tested and working
- [x] Unit tests

## Test plan
`make run_command "cmd=python src/manage.py populate_majors"`

### CODE REVIEW

**Comment by ramsgoli** [comment]
@marktai do you mean remove all migrations? even the unique=True migration?

File: src/users/migrations/0013_auto_20171206_2354.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        ('users', '0012_auto_20171206_1736'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='profile',
+            name='year',
+            field=models.CharField(choices=[('1st', '1st'), ('2nd', '2nd'), ('3rd', '3rd'), ('4th', '4th')], default='1st', max_length=15),
+        ),
+    ]
```
Comment (marktai) [inline src/users/migrations/0013_auto_20171206_2354.py]: just this one

**Comment by ramsgoli** [comment]
@marktai  done i think

**Comment by ramsgoli** [comment]
can i merge this?

**Comment by marktai** [comment]
it's approved :+1:

**Pull request merged**